services/outputhost/cgcache.go

```diff
@@ -500,9 +502,55 @@ func (cgCache *consumerGroupCache) refreshCgCache(ctx thrift.Context) error {
 		}
 	}
 
+	newDLQMerging := nSingleCGExtents > 0
+	if cgCache.dlqMerging != newDLQMerging {
+		cgCache.dlqMerging = newDLQMerging
+		if newDLQMerging {
+			cgCache.logger.WithField(`nSingleCGExtents`, nSingleCGExtents).Info("Merging DLQ Extent(s)")
+
+		} else {
+			cgCache.logger.Info("Done merging DLQ Extent(s)")
+		}
+	}
+
 	return nil
 }
 
+var checkSingleCGVisibleCacheLock sync.RWMutex
+var checkSingleCGVisibleCache = make(map[string]bool)
```

Comment:
Can you move these global static vars into cgCache ? Also don't like the fact that this cache has no eviction policy. Consider using the LruCache in common.